server/src/test/java/org/elasticsearch/VersionTests.java

```diff
@@ -346,7 +346,7 @@ public static void assertUnknownVersion(Version version) {
     public void testIsCompatible() {
         assertTrue(isCompatible(Version.CURRENT, Version.CURRENT.minimumCompatibilityVersion()));
         assertFalse(isCompatible(Version.V_6_6_0, Version.V_7_0_0));
```

Comment:
let's replace it with `assertFalse(isCompatible(Version.V_7_0_0, Version.V_8_0_0))` to please the tests, by removing the 6.8 version we made Elasticsearch believe that 6.6 is the latest 6.x version and thus that it is compatible with 7.0

Reply:
Thanks, this fixed the test.